packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -200,6 +381,69 @@ _CSS Classes and CSS Custom Properties that affect the component_
 
 _Note_: This initial component design serves as a starting point for implementation, and it may undergo changes once the visual design is completed.
 
+#### User mention element:
+
+This element serves as a container for the mapping elements and additionally functions as a configuration element. It allows the transmission of
+arbitrary values that are specific to the `@mention` users within the rich text components. Currently, this component do not have any
+configuration properties/attributes for user mentions and plays the role of containing element for the user mapping elements.
+
+_Component Name_
```

Comment:
Is there a way to get all the mentioned users? In the Skyline HLD there was a proposal to require the client to provide that list rather than have the service extract it from the markdown.

Reply:
Initially, we thought that the mentioned users to the backend services could be parsed from the markdown string in SLE, identify all the user IDs from the markdown string, and push it to an array to make the API call to backend services. Because we thought that for the viewer's API, we have to parse the string anyway in SLE but it seems the backend will give those mentioned users data to construct the necessary configuration elements for the viewer.

Therefore, we decided that it would be better to send it from the Nimble which handles all the markdown-related operations, so we planned to expose a read-only property called `mentioned-users` which will get the user IDs that are added to the editor with the help of `markdown-serializer`. This will return the user IDs in the form of an array. I have updated the spec to capture the same

Reply:
test